runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -2249,8 +2249,9 @@ private ReadWithIds(UnboundedSource<T, ?> source) {
       @Override
       public void populateDisplayData(DisplayData.Builder builder) {
         super.populateDisplayData(builder);
-        builder.add(DisplayData.item("source", source.getClass()));
-        builder.include(source);
+        builder
+            .add(DisplayData.item("source", source.getClass()))
+            .include("source", source);
```

Comment:
I just did a pass over existing usages to see how this would fit, and I like it. Some thoughts:

* Presumably the default behavior would be to .`add()` a display data item with the specified path and item class, then `.include()` it's display data. This covers most existing scenarios.
* Some components deviate from this convention, but in ways that we could by adding additional customizations to the `.nested()` builder:
  * Not registering the class item: `.withoutClassItem()`
  * Registering a different class item: `.withClass(unwrappedFn.getClass())`
  * Adding label metadata to the item: `.withLabel("..")`
* This should subsume `.include()` entirely to keep the API small. Between `.add()`, `.nested()`, and `.delegate()` all scenarios seem covered.

Let me separate that work out to a separate PR.